commit 3ce15aeaa66a2f523c6fa92bfe818734bdedfcea
Author: Noah Friedman <friedman@splode.com>
Date:   Thu Mar 15 12:52:50 2012 -0400

    rpc.idmap: Hide global symbols from libidmap plugins
    
    This patch limits the visibility of the symbols in the nfs-utils
    conffile.c so that they are only visible to programs linked directly to
    it.  This forces the objects dynamically loaded via libnfsidmap to use
    the functions defined in that shared library instead.
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/support/nfs/conffile.c b/support/nfs/conffile.c
index 2f1e235..5015e94 100644
--- a/support/nfs/conffile.c
+++ b/support/nfs/conffile.c
@@ -49,6 +49,8 @@
 #include "conffile.h"
 #include "xlog.h"
 
+#pragma GCC visibility push(hidden)
+
 static void conf_load_defaults(void);
 static int conf_set(int , char *, char *, char *, 
 	char *, int , int );
